manifests/m/Microsoft/EdgeWebView2Runtime/111.0.1661.54/Microsoft.EdgeWebView2Runtime.installer.yaml

```diff
@@ -0,0 +1,57 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.EdgeWebView2Runtime
+PackageVersion: 111.0.1661.54
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- silentWithProgress
+- silent
+InstallerSwitches:
+  Silent: /silent /install
+InstallerSuccessCodes:
+- 2147748109

```

Comment:
this exit code means there is a more recent version installed. Since Edge runtime uses a Evergreen update https://learn.microsoft.com/en-gb/microsoft-edge/webview2/concepts/distribution#the-evergreen-runtime-distribution-mode, it means that the installed version might be updated before we run the `winget upgrade` command. 

In that case, we should mark the install as successful for  the older version we are trying to install.